.github/workflows/workflow.yml

```diff
@@ -1,19 +1,29 @@
 name: main
 on:
-  push: { branches: [main] }
-  pull_request: { branches: [main] }
+  push: 
+    branches: [main]
+    tags: "v*"
+  pull_request:
+    branches: [main]
 concurrency:
   group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
   cancel-in-progress: true
+env:
+  BUILD_DEB_OUTPUT_DIR: out
+  BUILD_PYTHON_OUTPUT_DIR: dist
+  # The latest version supporting Python 3.6
+  POETRY_VERSION: 1.1.15
```

Comment:
Don't we want to rely on poetry version defined in Makefile and keep it only in one place?

Reply:
Agree. I left POETRY_VERSION only in makefile.